torch/cuda/amp/grad_scaler.py

```diff
@@ -279,7 +281,7 @@ def unscale_(self, optimizer):
         # FP32 division can be imprecise for certain compile options, so we carry out the reciprocal in FP64.
         assert self._scale is not None
         inv_scale = self._scale.double().reciprocal().float()
-        found_inf = torch.full((1,), 0.0, dtype=torch.float32, device=self._scale.device)
+        found_inf = torch.zeros((), dtype=torch.float32, device=self._scale.device)
```

Comment:
Let's be consistent here choose whether we want to use .full or .tensor everywhere.
```suggestion
        found_inf = torch.full((), 0.0, dtype=torch.float32, device=self._scale.device)
```